app/renderer/controls/Button.tsx

```diff
@@ -9,7 +9,7 @@ type StyledButtonProps = {
   isDisabled?: boolean;
 };
 
-const StyledButton = styled.span<StyledButtonProps>`
+const StyledButton = styled.button<StyledButtonProps>`
```

Comment:
Can you add style for the focus outline? I have defined a new focused style for the button:

![State=Focused](https://user-images.githubusercontent.com/6120081/193588130-7803ca1a-fc9e-4b4d-8ee9-86de032ba8e0.png)

It is a 4px solid outline using theme.buttonPrimaryBg at 50% alpha. You can use `Color(theme.buttonPrimaryBg).alpha(0.5).toString()` to acheive this.

Be sure to use the `focus-visible` psuedo-class instead of just `focus` to prevent the outline from being shown when interacting with the button via mouse.

Reply:
Certainly! The requested change has been added to this PR.